src/main/webapp/i18n/en/notification.json

```diff
@@ -52,6 +52,8 @@
                 "programmingTestCasesChanged": "Test cases for programming exercise changed",
                 "newManualFeedbackRequest": "A new manual feedback request has been submitted",
                 "duplicateTestCase": "Duplicate test case was found.",
+                "repositoryLocks": "Permissions Changed",
+                "buildRun": "Build Run Update",
```

Comment:
The other strings don't use title case here, is that on purpose?

Reply:
I'd prefer the titles to be consistent with their capitalization.
Maybe we should think about shortening all of them and putting important info into the text instead the title